apps/developer-portal/src/components/shared/advanced-search-with-basic-filters.tsx

```diff
@@ -234,7 +241,7 @@ export const AdvancedSearchWithBasicFilters: FunctionComponent<AdvancedSearchWit
                                         };
                                     })
                                 }
-                                readOnly={ filterAttributeOptions.length === 1 }
+                                // readOnly={ filterAttributeOptions.length === 1 }
```

Comment:
Shall we add a TODO here? Else we'll remove it if we are not going to enable back in the future

Reply:
Fixed with `4f3eb92`